packages/react-native/ReactCommon/yoga/yoga/algorithm/CalculateLayout.cpp

```diff
@@ -2140,7 +2140,8 @@ bool calculateLayoutInternal(
 
```

Comment:
These files will ultimately get synced everywhere, but we usually prefer changes against Yoga to be made against the Yoga repo (which will run some additional validation in OSS as well).

Yoga is also pretty heavily unit tested, so we would want to add a test alongside this. Unit tests are run automatically against the Yoga repo, but there is also a `unit_tests.bat` that will build and run the tests on Windows/MSVC so long as CMake is set up correctly (I think you can run from VS command shell)?